modules/runtime/src/home/config/com.enonic.xp.project.cfg

```diff
@@ -0,0 +1 @@
+#multipleParents.allowed = false
```

Comment:
this is a "secret" config. we must not make it visible for now.
Delete the file